Vungle/VungleAdapter/ALVungleMediationAdapter.m

```diff
@@ -7,20 +7,51 @@
+
+@interface ALVungleMediationInterstitialAdapterRouter : NSObject<VungleInterstitialDelegate>
+@property (nonatomic, weak) ALVungleMediationAdapter *parentAdapter;
+@property (nonatomic, strong) id<MAInterstitialAdapterDelegate> interstitialAdDelegate;
+- (nonnull instancetype)initVungleInterstitialAdDelegate:(id<MAInterstitialAdapterDelegate>)interstitialAdDelegate parentAdapter:(ALVungleMediationAdapter *)parentAdapter;
+@end
+
+@interface ALVungleMediationRewardedAdapterRouter : NSObject<VungleRewardedDelegate>
+@property (nonatomic, weak) ALVungleMediationAdapter *parentAdapter;
 @property (nonatomic, assign, getter=hasGrantedReward) BOOL grantedReward;
-@property (nonatomic, strong) NSMutableDictionary<NSString *, NSString *> *creativeIdentifiers;
+@property (nonatomic, strong) id<MARewardedAdapterDelegate> rewardedAdDelegate;
+- (nonnull instancetype)initVungleRewardedAdDelegate:(id<MARewardedAdapterDelegate>)rewardedAdDelegate parentAdapter:(ALVungleMediationAdapter *)parentAdapter;
+@end
 
-- (void)updateUserPrivacySettingsForParameters:(id<MAAdapterParameters>)parameters consentDialogState:(ALConsentDialogState)consentDialogState;
-- (nullable NSNumber *)privacySettingForSelector:(SEL)selector fromParameters:(id<MAAdapterParameters>)parameters;
+@interface ALVungleMediationAdViewAdapterRouter : NSObject<VungleBannerDelegate>
+@property (nonatomic, weak) ALVungleMediationAdapter *parentAdapter;
+@property (nonatomic, strong) UIView *adView;
+@property (nonatomic, strong) VungleBanner *vungleBannerAd;
+@property (nonatomic, strong) MAAdFormat *adFormat;
+@property (nonatomic, strong) id<MAAdViewAdapterDelegate> adViewAdDelegate;
+@property (nonatomic, strong) id<MAAdapterResponseParameters> parameters;
+- (nonnull instancetype)initVungleAdViewAdDelegate:(id<MAAdViewAdapterDelegate>)adViewAdDelegate parentAdapter:(ALVungleMediationAdapter *)parentAdapter parameters:(id<MAAdapterResponseParameters>)parameters adFormat:(MAAdFormat *)adFormat;
+- (void)loadAdView:(NSString *)placementIdentifier;
+- (void)destroy;
+@end
+
+@interface ALVungleMediationNativeAdAdapter : NSObject<VungleNativeDelegate>
+@property (nonatomic, strong) VungleNative *vungleNativeAd;
+@property (nonatomic, strong) id<MANativeAdAdapterDelegate> nativeAdDelegate;
+@property (nonatomic, strong) id<MAAdapterResponseParameters> parameters;
+- (nonnull instancetype)initVungleNativeAdDelegate:(id<MANativeAdAdapterDelegate>)nativeAdDelegate parameters:(id<MAAdapterResponseParameters>)parameters;
+- (void)requestNativeAd:(NSString *)placementIdentifier;
+- (void)unregisterNativeAd;
+- (void)destroy;
+@end
+
+@interface MAVungleNativeAd : MANativeAd
+@property (nonatomic, weak) ALVungleMediationNativeAdAdapter *parentAdapter;
+- (instancetype)initWithParentAdapter:(ALVungleMediationNativeAdAdapter *)parentAdapter builderBlock:(NS_NOESCAPE MANativeAdBuilderBlock)builderBlock;
 @end
 
 @interface ALVungleMediationNativeAdAdapter : NSObject<VungleNativeAdDelegate>
```

Comment:
We need to revert `Merge branch '7.x_VungleSDK' into Vungle_7.0`, because we have duplicated definitions:
`@interface ALVungleMediationNativeAdAdapter : NSObject<VungleNativeDelegate>`  vs 
`@interface ALVungleMediationNativeAdAdapter : NSObject<VungleNativeAdDelegate>`
Thanks a lot. :)

Reply:
good catch. thanks. i missed it while resolving conflicts.